packages/backend/src/rates/service.ts

```diff
@@ -96,23 +96,23 @@ class RatesServiceImpl implements RatesService {
       }
     }
 
-    if (!this.pricesRequest) {
-      this.pricesRequest =
+    if (!this.ratesRequest) {
+      this.ratesRequest =
         this.prefetchRequest ||
         this.loadNow().catch((err) => {
-          this.pricesRequest = undefined
-          this.pricesExpiry = undefined
+          this.ratesRequest = undefined
+          this.ratesExpiry = undefined
           throw err
         })
     }
-    return this.pricesRequest
+    return this.ratesRequest
   }
 
-  private async loadNow(): Promise<Prices> {
-    const url = this.deps.pricesUrl
+  private async loadNow(): Promise<Rates> {
+    const url = this.deps.exchangeRatesUrl
     if (!url) return {}
```

Comment:
when no url, should we return 
```ts
    const data = {
      [base]: 1.0,
      rates: {} 
    }
```
?

Reply:
Shouldn't it be just:
```ts
const data = {
  [base]: 1.0,
}
```
?

Reply:
@raducristianpopa is right, but it is not necessary. If `{}` is returned, `{ [base]: 1.0 }` is assumed by the pay library (which I think is the only code that consumes that response, but I'm still digging through the code to understand that part better).